packages/anyrender_vello/src/custom_paint_source.rs

```diff
@@ -53,32 +52,29 @@ impl CustomPaintCtx<'_> {
             origin: wgpu::Origin3d::ZERO,
             aspect: wgpu::TextureAspect::All,
         };
-        self.renderer.override_image(&dummy_image, Some(base));
+        self.renderer.override_image(&dummy_image.image, Some(base));
```

Comment:
This should use the new `register_texture` API:

```rust
let dummy_image = self.renderer.register_texture(texture);
```

And TextureHandle should become a newtype around `peniko::ImageData`.